site/layouts/pricing/list.html

```diff
@@ -13,157 +13,138 @@ <h1 class="display-3">Flexible pricing</h1>
+                <div class="pricing__fancy-price__price pricing__fancy-price__price--big pricing__fancy-price__price--white">1,500</div>
               </div>
+              <div>per month</div>
             </div>
             <div class="pricing__btn__wrapper">
-              <a class="btn btn-success pricing__btn mb-3" href="https://app.checklyhq.com/signup" rel="noopener">Try for free</a>
-              <small class="text-muted text-left align-self-start mb-5"> * No credit card required.</small>
-            </div>  
-            <div class="pricing__list pricing__list--team">
-              <ul class="pt-3">
+              <button type="button" class="btn btn-secondary pricing__btn__secondary pricing__btn mb-3" data-toggle="modal" data-target="#contact-sales-modal">
+                Contact sales
+              </button>
+            </div>
+            <div class="pricing__list mt-5">
+              <ul class="pt-5 mx-3">
                 <li class="font-weight-bold">
-                  <i class="ion-ios-checkmark text-primary mr-2"></i>
-                  All features on the Hobby plan
+                  Everything in the Team plan plus:
                 </li>
-                <li>
+                <li class="align-items-center">
                   <i class="ion-ios-checkmark text-primary mr-2"></i>
-                  Includes check runs worth $30
+                  Custom amount of check runs
                 </li>
-                <li>
-                  <i class="ion-ios-checkmark text-primary mr-2"></i>
-                  20 Users
-                </li>
-                <li>
-                  <i class="ion-ios-checkmark text-primary mr-2"></i>
-                  Unlimited Browser and API checks
-                </li>
-                <li>
+                <li class="align-items-center">
                   <i class="ion-ios-checkmark text-primary mr-2"></i>
-                  1 year of aggregated data retention
+                  Unlimited Users
                 </li>
-                <li>
+                <li class="align-items-center">
                   <i class="ion-ios-checkmark text-primary mr-2"></i>
-                  Prometheus endpoint
+                  99.9% SLA!
                 </li>
-                <li>
+                <li class="align-items-center">
                   <i class="ion-ios-checkmark text-primary mr-2"></i>
-                  All alert channels
+                  25 months of aggregated data retention
                 </li>
                 <li>
-                  <i class="ion-ios-checkmark text-primary mr-2"></i>
-                  Pro support
+                  + SSO/SAML, Private locations, Client certificates, premium support, and more.
                 </li>
+                <li class="mt-4">
+                  Build and run synthetic monitoring across distributed teams and systems with an advanced MaC workflow delivering programmability, speed, resilience.                </li>
```

Comment:
What about adding some sort of accent to this text? Making it bold or so, it looks sort of muted. Maybe @jenya-II might have ideas
![image](https://user-images.githubusercontent.com/25155364/216114043-8cafab8e-422d-4fc3-b3c4-681ecd579bc6.png)

Reply:
Yes, that may work. Let's see what @jenya-II thinks because I did this following [her design] 😂(https://www.figma.com/file/et2KLDMVOPb0xeAmXFJXok/checklyhq.com?node-id=2661%3A12380&t=lx6tnTKEfdd3mPa4-0)

Reply:
@pilimartinez @shiini2 i would stick to simpler solution for now, we are changing these texts quite often, so i would not overdesign them